### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by s58237**
Issue #220 were implemented.

Issue #221 and #222 were realized and other libaries were updated.

Settings.properties and Settings_mc.properties file were merged in one file ("settings.properties) and the keys were summerized.

examples/stdAnalysis/data/analysis.xml and examples/stdAnalysis/mc/analysis_mc.xml were merged in one file "analysis.xml". The characteristic "fTotalEnergy" in the data deciedes wheter a data or an mc-analysis is executed.When fTotatlEnergy==null then a data-analysis is choosen.

Code of xml-files in src/main/resources/default is inserted directly in files of example an the xml files werde deleted.

### CODE REVIEW

File: src/main/java/fact/auxservice/AuxPoint.java [hunk 1]
```diff
@@ -12,11 +13,11 @@
...
  */
 public class AuxPoint implements Comparable<AuxPoint>{
-    private DateTime timeStamp;
+    private OffsetDateTime timeStamp;
     private ImmutableMap<String, Serializable> data;
 
-    public AuxPoint(DateTime timeStamp){
-        this.timeStamp = new DateTime(timeStamp);
+    public AuxPoint(OffsetDateTime timeStamp){
+        this.timeStamp =OffsetDateTime.of(timeStamp.toLocalDate(),timeStamp.toLocalTime(), ZoneOffset.of("+00:00"));
```
Comment (mackaiver) [inline src/main/java/fact/auxservice/AuxPoint.java]: Why is it neccessary to create a new object here? It's basically a copy isn't it?

File: src/main/java/fact/auxservice/AuxPoint.java [hunk 2]
```diff
@@ -25,17 +26,17 @@ public AuxPoint(DateTime timeStamp){
      * @param timeStamp the timestamp specifying  when the auxiliary data was recorded.
      * @param data a map containing the names and values from the .fits files as key values pairs.
      */
-    public AuxPoint(DateTime timeStamp, Map<String, Serializable> data){
+    public AuxPoint(OffsetDateTime timeStamp, Map<String, Serializable> data){
         this.data = new ImmutableMap.Builder<String, Serializable>().putAll(data).build();
-        this.timeStamp = new DateTime(timeStamp);
+        this.timeStamp =OffsetDateTime.of(timeStamp.toLocalDate(),timeStamp.toLocalTime(), ZoneOffset.of("+00:00"));
```
Comment (mackaiver) [inline src/main/java/fact/auxservice/AuxPoint.java]: see above

**Comment by MaxNoe** [comment]
I think this is too large to be reviewable and too much can break.

Can we have only the library concerned changes first? These are important to get fast into master and should be easy to review. 

Also today we spoke about the question `OffsetDateTime` vs `ZonedDateTime` and concluded that we should use `ZonedDateTime` everywhere which is just minor changes.

**Comment by MaxNoe** [comment]
I opened a PR with only the library changes

**Comment by MaxNoe** [comment]
I don't like that these containers with the standard calibration and extraction stuff got removed. I like these building blocks for ease of use.

**Comment by MaxNoe** [comment]
I'm closing this for now as there is too much to be effectively reviewed and to much conflicts to the master.

**Pull request closed without merging**